src/annotator/util/buckets.ts

```diff
@@ -198,10 +139,10 @@ export function computeBuckets(
     const center = (aPos.top + aPos.bottom) / 2;
 
     if (center < vMargin) {
-      aboveTags.add(aPos.tag);
+      aboveAnchors.push(aPos);
```

Comment:
The anchor we are actually adding to the bucket is the one that has had coordinates translated to be relative to the bucket bar, rather than the inputs which were viewport-relative. That's actually fine given how we later use these anchors, which is to sort and compare them. It may or may not be appropriate for other uses.